src/findingApi.js

```diff
@@ -15,8 +15,12 @@ const findItemsByKeywords = function (options) {
     this.options.param = 'keywords';
     // support only keyword string.
     if (!options.keywords) {
-        this.options.name = options;
+        this.options.name = encodeURIComponent(options);
     }
+    else {
+        options.keywords = encodeURIComponent(options.keywords);
+    }
+
     this.options.additionalParam = constructAdditionalParams(options);
```

Comment:
@abbadata  Thanks for your contribution. We can encode keyword using encodeURIComponent

```javascript
if (!options.keywords) options = { "keywords": options };
options.keywords = encodeURIComponent(options.keywords);
```

Reply:
I checked in your suggested fix. Thanks.

Reply:
@abbadata Thanks 👍